manifests/master/tuning.pp

```diff
@@ -26,26 +26,24 @@
 
     case $jvm_tuning_profile {
       'lvm': {
-        $amq_heap_mb                = '96'
```

Comment:
Did you mean to change the LVM profile to these parameters or to create a new profile?